src/components/my-map/configuration/evt-widget-config-marker-color/evt-widget-config-marker-color.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+import './evt-widget-config-marker-color.scss';
+
+/**
+ * Configuration component example
+ * Allows to select color to be used for markers on the map
+ */
+class EvtWidgetConfigMarkerColor {
+
+  /**
+   * Setup injection and initialize component variables here.
+   *
+   * @param EvtColorCategory Editor allows to select colors from EVT Color Schema
+   */
+  constructor(EvtColorCategory) {
+    "ngInject";
+
+    this.EvtColorCategory = EvtColorCategory;
+  }
+
+  /**
+   * Run initialization code here.
+   * See component lifecycle hooks:
+   * https://docs.angularjs.org/guide/component
+   *
+   * Initialises colors model to work with
+   * If previously selected color wasn't found - selects first one available
+   */
+  $onInit() {
+    /**
+     * Defines model of colors to allow selection from
+     * @type {Color[]}
+     */
+    this.colors = [
+      ...this.EvtColorCategory.primaryPalette,
+      ...this.EvtColorCategory.grayScalePalette
+    ].map(this.hexToColor, this);
+
+    // If no colors selected
+    if (!this.colors.some(color => color.selected)) {
+      this.select(this.colors[0]);
+    }
+  }
+
+  /**
+   * Selects the color, updates model and triggers change expression binding
+   *
+   * @param {Color} color
+   */
+  select(color) {
+    if (!this.isDisabled) {
+      this.colors.forEach(this.deselect);
+
+      color.selected = true;
+
+      this.model.config.value = color.value;
+      this.onChange();
+    }
+  }
+
+  /**
+   * Deselects given color
+   *
+   * @param {Color} color
+   */
+  deselect(color) {
+    color.selected = false;
+  }
+
+  /**
+   * @typedef {object} Color
+   * @property {number} height - defines md-whiteframe height of color card
+   * @property {string} value - real color value
+   * @property {boolean} selected - whether this color selected or not
+   */
+
+  /**
+   * Converts given hex string to color model
+   *
+   * @param {string} hex
+   * @return {Color}
+   */
+  hexToColor(hex) {
+    const value = hex.replace('#', '');
+
+    return {
+      height: 2,
+      value: value,
+      selected: this.isModelValue(value)
+    };
+  }
+
+  /**
+   * Returns true if given hex value equals to real model value
+   *
+   * @param {string} hex
+   * @return {boolean}
+   */
+  isModelValue(hex) {
+    console.log(this.model.config);
+    return this.model.config.value === hex;
+  }
+}
+
+angular.module('myModule.components.configuration.evtWidgetConfigMarkerColor', [])
```

Comment:
Lots of comments on same subject. I'll address all them here, since this repo was supposed to be as simple as possible i decided to move configuration component out of main folder so noone confused with its name.

We can consider fully custom name for configuration editor components. I was thinking about this already. Until then we can use this, not a big deal to fix compared to up-to-date examples.